README.md

```diff
@@ -75,12 +75,28 @@ Scribble Stadium is a paid service; parents are required to create the account f
 |                [<img src="https://avatars.githubusercontent.com/u/61174160?v=4" width = "100" />](https://github.com/Qirhi)                 |            [<img src="https://avatars.githubusercontent.com/u/21160325?v=4" width = "100" />](https://github.com/mollybee)             |            [<img src="https://avatars.githubusercontent.com/u/72050425?v=4" width = "100" />](https://github.com/Ksouffle)            |               [<img src="https://avatars.githubusercontent.com/u/64991544?v=4" width = "100" />](https://github.com/milklot)                |           [<img src="https://avatars.githubusercontent.com/u/52305213?v=4" width = "100" />](https://github.com/Julianisrel)           |     [<img src="https://avatars.githubusercontent.com/u/69640436?v=4" width = "100" />](https://github.com/bporch84)     |
-|                                                                     Web                                                                     |                                                                  Web                                                                   |                                                                  Web                                                                  |                                                                     Web                                                                     |                                                                  Web                                                                   |                                                           Web                                                           |
+|                                                                     Technical Lead Engineer                                                                     |                                                                  Team Project Lead                                                                   |                                                                  Lead Backend Engineer                                                                  |                                                                     Frontend Engineer                                                                     |                                                                  UX Engineer                                                                   |                                                           Lead UX Engineer                                                           |
```

Comment:
Thanks for catching this. I know I changed this myself; I even sent you a snapshot, but with all the copying and pasting I was doing, I must have gotten mixed up somewhere.

Reply:
You did great.  I am Always forgetting to save the file.  I do this all the time.